book.rb

```diff
@@ -0,0 +1,14 @@
+# Defines the Book class blueprint
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author
```

Comment:
### Optional
- [ ] I recommend you to set the "attr_accesor" at the beginning of the class.